cnf-certification-test/identifiers/identifiers.go

```diff
@@ -600,6 +605,17 @@ test case requires the Deployment of the debug daemonset.`),
 		ExceptionProcess:      NoDocumentedProcess,
 	},
 
+	TestServiceDualStackIdentifier: {
+		Identifier:  TestServiceDualStackIdentifier,
+		Type:        normativeResult,
+		Remediation: TestServiceDualStackRemediation,
+		Description: formDescription(TestServiceDualStackIdentifier,
+			`Checks that all services in namespaces under test are either ipv6 single stack or dual stack . This
```

Comment:
Remove whitespace after "stack". Why capital "D" for "Deployment"?

Reply:
fixed